src/if_statements.py

```diff
@@ -167,6 +234,25 @@ def get_lone_if_statement(self) -> Optional[IfElseStatement]:
                 ret = statement
         return ret
 
+    def elide_empty_returns(self) -> None:
```

Comment:
should we expand this to also elide duplicate returns and/or warn about unreachable post-return statements?

Reply:
actually I think this is more appropriate for the `emit_node` discussion

Reply:
I think this refactor addressed those issues we were having before where we would sometimes output `return; return; return;` (ex `func_80B8E84C` in MM) -- that stopped happening even before I added `elide_empty_returns`.